sp_api/api/reports/reports.py

```diff
@@ -167,8 +167,9 @@ def get_report(self, reportId, **kwargs) -> ApiResponse:
             ApiResponse
 
         """
-
-        return self._request(fill_query_params(kwargs.pop('path'), reportId), params=kwargs)
+        add_marketplace = kwargs.pop("add_marketplace") if "add_marketplace" in kwargs else False
```

Comment:
```suggestion
        add_marketplace = kwargs.pop('add_marketplace') if 'add_marketplace' in kwargs else False
```
let's try to keep the code style consistent